pkgs/development/libraries/libngspice/default.nix

```
@@ -21,6 +21,6 @@ stdenv.mkDerivation rec {
     homepage = "http://ngspice.sourceforge.net";
     license = with licenses; [ bsd3 gpl2Plus lgpl2Plus ]; # See https://sourceforge.net/p/ngspice/ngspice/ci/master/tree/COPYING
     maintainers = with maintainers; [ bgamari ];
-    platforms = platforms.linux;
+    platforms = with platforms; darwin ++ linux;
```

```suggestion
    platforms = with platforms.unix;
```

Simpler and is likely to work with any unix.

---

Thanks for the swift reply and improvement suggestion. I'm new to nix always keen to learn how things are done in an idiomatic way.
Change applied to PR.